data/test/scenarios/wml_tests/UnitsWML/unit_no_advancements.cfg

```diff
@@ -0,0 +1,69 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [unit] [advancement]
```

Comment:
The current standard is to have less spacing on the "API(s) being tested:" line - please do a `git grep` and see how the others look.